Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3279,20 +3276,21 @@ - (void)endRecordingVoiceMemo
     }
 }
 
+- (void)stopRecording
+{
+    [self.audioRecorder stop];
+    [OWSAudioSession.shared endAudioActivity];
+}
+
 - (void)cancelRecordingVoiceMemo
 {
     OWSAssertIsOnMainThread();
 
     DDLogDebug(@"cancelRecordingVoiceMemo");
 
-    [self resetRecordingVoiceMemo];
-}
-
-- (void)resetRecordingVoiceMemo
-{
```

Comment:
Huh, isn't this `OWSAssertIsOnMainThread` twice in a row?

Reply:
Removed the redundant assert - thanks!